apis/sgv2-restapi/src/test/java/io/stargate/sgv2/it/RestApiV2QCqlIT.java

```diff
@@ -6,6 +6,10 @@
 import io.quarkus.test.junit.QuarkusIntegrationTest;
 import io.restassured.http.ContentType;
 import io.stargate.sgv2.common.testresource.StargateTestResource;
+import java.util.Arrays;
+import org.apache.groovy.util.Maps;
```

Comment:
strange import, should you use `Map.of()` (java Map?)

Reply:
Ouch. Good catch, no that's wrong.